indexer/src/test/java/bio/terra/tanagra/indexing/JobSequencerTest.java

```diff
@@ -151,6 +152,36 @@ public void conditionPerson() {
     assertEquals(2, jobStageItr.next().size());
   }
 
+  @Test
+  public void conditionPersonCleanHierarchyNodesWithZeroCounts() {
+    ConfigReader configReader = ConfigReader.fromJarResources();
+    SZIndexer szIndexer = configReader.readIndexer("aouSR2019q4r4_verily");
```

Comment:
nit: Use `aouSR2019q4r4_broad` instead of the `_verily` version. I try to use the `broad` config files wherever possible in tests because those are the only ones the GHA has permissions to query. Technically in these `JobSequencer` tests it doesn't matter, since we're not running any of the jobs, just checking how they're sequenced, but I think it would be good for consistency.

Reply:
Switched to use one of our indexer json files: aou/SC2023Q3R2